.changelog/1705.trivial.md

```diff
@@ -0,0 +1 @@
+Add support for displaying multiple signers of runtime transactions
```

Comment:
I would mark this as a feature, not trivial

Reply:
OK, renamed as feature.